tools/amGithubNotifier/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM quay.io/prometheus/busybox-${OS}-${ARCH}:latest
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
+
+COPY amGithubNotifier /bin/amGithubNotifier
```

Comment:
I prefer if we could build the image without relying on make.
Using docker multistage builds.

And can you check if the difference between using busybox and scratch is big than can use scracth image, unless there is any other reason to use busybox.

Reply:
when building the image with scratch the image size(21MB) is 3MB less than that one with busybox(24MB). should I make the change?

Reply:
no need, but make the binary build inside the Dockerfile using multistage builds.